src/lib/Libifl/pbs_loadconf.c

```diff
@@ -281,6 +281,9 @@ parse_psi(char *conf_value)
 	free(pbs_conf.psi_str);
 
 	if (conf_value == NULL) {
+		char *dfltsvr = pbs_default();
```

Comment:
i don't see how this fixes a leak. Or even valgrind report is correct (valgrind shows spurious leaks all the time). 
More importantly, we need to add code to prevent reloading of the PBS_SERVER_INSTANCES variable. That should only be loaded once at daemon startup

Reply:
I'm not sure either, but it's a good check to have anyways. I've added code to prevent re-read of PBS_SERVER_INSTANCES on a reload, thanks for catching that.